### PR TITLE
Made required changes as per suggestions

### LINKED ISSUE #48: UI for code-of-conduct page
1. Follow CSS BEM architecture. 
2. Make a separate SCSS file for this page. 
3. Do not compile scss.

### PR DESCRIPTION
**Pull request by shwetalsoni**
Fixes #48 
Changed the images names according to their usage
card1, card2, card3 renamed as card
block1, block2 and block3 changed to guidelines-card
hero_section

code_of_conduct
guidelines

### CODE REVIEW

File: frontend/src/components/CodeOfConduct.jsx [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+                            <img src={CodeofConduct1} alt="CodeofConduct1"  className="block1__image"/>
+
+                            <p className="block1__text">Equity is essential because it allows others to treat individual wants respectfully</p>
+
+                        </div>
+                    </div>
+
+                    <div className="cell large-4">
+                        <div class="block2">
```
Comment (salil-naik) [inline frontend/src/components/CodeOfConduct.jsx:108]: replace block1, block2, etc with `guidelines-card`

The classes can be 
`guidelines-card__image`
`guidelines-card__text`

**Review by salil-naik — CHANGES_REQUESTED** [review]
Kindly do the changes. Do not use npm anywhere.

File: frontend/src/components/CodeOfConduct.jsx [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+           <h2 className="main-content__heading"> Code of Conduct</h2>
+
+            <div className="grid-container">
+                <div className="grid-x grid-margin-x">
+
+                   
+                        <div className="cell large-6">
+
+                            <div className="card1">
```
[the remark this answers is not in the document]
Comment (shwetalsoni) [reply frontend/src/components/CodeOfConduct.jsx:43]: Here , I have given it different classes as i had to give it margin ..so if use just "card" i wont be able to do that.

File: frontend/src/components/CodeOfConduct.jsx [hunk 3]
```diff
@@ -0,0 +1,138 @@
+import React, { Component } from "react";
+import photo from "../images/download 1.png";
+import '../css/components/CodeofConduct.scss';
+import CodeofConduct1 from "../images/CodeofConduct1.png";
```
[the remark this answers is not in the document]
Comment (shwetalsoni) [reply frontend/src/components/CodeOfConduct.jsx:4]: yeah , did that

File: frontend/src/components/CodeOfConduct.jsx (hunk 1 above)
Comment (shwetalsoni) [reply frontend/src/components/CodeOfConduct.jsx:108]: done

File: frontend/src/components/CodeOfConduct.jsx [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+
+
+class CodeOfConduct extends Component {
+    render() {
+      return (
+
+    <div className="CodeofConduct">
+
+        <div className="hero-section">
+            <div className=" grid-container top">
```
Comment (salil-naik) [inline frontend/src/components/CodeOfConduct.jsx:16]: - [ ] remove unused class
- [ ] Use prettier extension to format jsx code

**Review by salil-naik — CHANGES_REQUESTED** [review]

**Review by salil-naik — APPROVED** [review]

**Pull request merged**